[2023-11-07T17:54:14.540+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract.dbt_run manual__2023-11-07T17:32:08.537403+00:00 [queued]>
[2023-11-07T17:54:14.552+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract.dbt_run manual__2023-11-07T17:32:08.537403+00:00 [queued]>
[2023-11-07T17:54:14.552+0000] {taskinstance.py:1361} INFO - Starting attempt 5 of 5
[2023-11-07T17:54:14.569+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): dbt_run> on 2023-11-07 17:32:08.537403+00:00
[2023-11-07T17:54:14.580+0000] {standard_task_runner.py:57} INFO - Started process 510 to run task
[2023-11-07T17:54:14.586+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'extract', 'dbt_run', 'manual__2023-11-07T17:32:08.537403+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/load.py', '--cfg-path', '/tmp/tmpdi_69h3c']
[2023-11-07T17:54:14.590+0000] {standard_task_runner.py:85} INFO - Job 8: Subtask dbt_run
[2023-11-07T17:54:14.675+0000] {task_command.py:416} INFO - Running <TaskInstance: extract.dbt_run manual__2023-11-07T17:32:08.537403+00:00 [running]> on host 48017e158481
[2023-11-07T17:54:14.836+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='extract' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2023-11-07T17:32:08.537403+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-07T17:32:08.537403+00:00'
[2023-11-07T17:54:14.840+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-07T17:54:14.841+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'dbt run --models first_model --project-dir /home/marius.fall/NBA_Statistics/nba_stats && echo ok']
[2023-11-07T17:54:14.858+0000] {subprocess.py:86} INFO - Output:
[2023-11-07T17:54:18.668+0000] {subprocess.py:93} INFO - Usage: dbt run [OPTIONS]
[2023-11-07T17:54:18.668+0000] {subprocess.py:93} INFO - Try 'dbt run -h' for help.
[2023-11-07T17:54:18.669+0000] {subprocess.py:93} INFO - 
[2023-11-07T17:54:18.669+0000] {subprocess.py:93} INFO - Error: Invalid value for '--project-dir': Path '/home/marius.fall/NBA_Statistics/nba_stats' does not exist.
[2023-11-07T17:54:18.981+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-07T17:54:19.018+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/root/.cache/pypoetry/virtualenvs/02-dbt-in-airflow-BsnH11MF-py3.8/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-07T17:54:19.023+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=extract, task_id=dbt_run, execution_date=20231107T173208, start_date=20231107T175414, end_date=20231107T175419
[2023-11-07T17:54:19.041+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 8 for task dbt_run (Bash command failed. The command returned a non-zero exit code 2.; 510)
[2023-11-07T17:54:19.057+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-07T17:54:19.088+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
